packages/template-retail-react-app/app/pages/account/orders.test.js

```diff
@@ -15,6 +15,14 @@ import useCustomer from '../../commerce-api/hooks/useCustomer'
 import Orders from './orders'
 import mockConfig from '../../../config/mocks/default'
 
+jest.mock('commerce-sdk-react-preview', () => {
+    const originalModule = jest.requireActual('commerce-sdk-react-preview')
+    return {
+        ...originalModule,
+        useCustomerBaskets: jest.fn().mockReturnValue({data: {baskets: [{currency: 'GBP'}]}})
+    }
+})
```

Comment:
Is this intentional to move away from nock and use module mocks? It looks to me that the nock mocking strategy is not working for this test suite (?).I'd like to understand the why.

Reply:
So for this test file I was able to replace the module mock with the corresponding msw handler, but there are some test files that the msw handler is not sufficient. I am looking into understanding why

Reply:
@kevinxh There are 4 test files that rely on mocking the module to mock baskets data in the template (that is returned from the `useCurrentBasket()` hook - the msw handlers do not intercept the network call to return data... I am confused by this because other test files are able to handle this just fine